### ISSUE TITLE
Build failure with websockets 0.11.*

### ISSUE THREAD

**Issue by bergmark** [eskimor/servant-subscriber#14]
```
src/Servant/Subscriber/Client.hs:87:9: error:
    • The constructor ‘WS.Text’ should have 2 arguments, but has been given 1
    • In the pattern: WS.Text bs
      In a case alternative: WS.Text bs -> return $ decode bs
      In a stmt of a 'do' block:
        case msg of {
          WS.Text bs -> return $ decode bs
          WS.Binary _
            -> error "Sorry - binary connections currently unsupported!" }
```

I published a trustee revision for 0.6.0.0 to make it buildable again (https://hackage.haskell.org/package/servant-subscriber-0.6.0.0/revisions/). So as I see it the next step would be to publish a new servant-subscriber release that supports websockets 0.11.

Cheers

**Comment by eskimor** [comment]
Thank you! I am currently on vacation - I will tackle this asap.

**Comment by eskimor** [comment]
Fixed in version 0.6.0.1
